viewer/packages/pointclouds/src/styling/AnnotationListPointCloudObjectCollection.ts

```diff
@@ -0,0 +1,27 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+
+import { PointCloudObjectCollection } from './PointCloudObjectCollection';
+
+/**
+ * A simple PointCloudObjectCollection that consists of an explicitly provided list of annotation IDs
+ */
+export class AnnotationIdPointCloudObjectCollection extends PointCloudObjectCollection {
+  private readonly _annotationIds = new Set<number>();
+
+  constructor(ids: number[]) {
+    super();
+    for (const id of ids) {
+      this._annotationIds.add(id);
+    }
+  }
```

Comment:
```suggestion
  constructor(ids: Iterable<number>) {
    super();
    this._annotations = new Set<number>(ids);
  }
```
should work and be more generic